src/core.js

```diff
@@ -1680,11 +1680,17 @@ define([
                 isCopyable: !multiselect && mug.options.isCopyable
             }));
         $baseToolbar.find('.fd-button-remove').click(function () {
-            var mugs = _this.getCurrentlySelectedMug(true);
-            if (mugs.length > 1 || (mugs.length && form.getChildren(mugs[0]).length)) {
+            var mugs = _this.getCurrentlySelectedMug(true),
+                errorMessage;
+            if (mugs.length > 1) {
+                errorMessage = "You are about to delete multiple questions. This cannot be undone";
+            } else if (mugs.length && form.getChildren(mugs[0]).length) {
+                errorMessage = "You are about to delete a question with child nodes. This cannot be undone";
```

Comment:
I'm curious about the reasoning behind this change.

Unfortunately [users don't read](http://www.joelonsoftware.com/uibook/chapters/fog0000000062.html). What do you think about simplifying to a single short error message: "Deleting multiple questions cannot be undone."